specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/examples/threatintelligence/AppendTagsThreatIntelligence.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "api-version": "2019-01-01-preview",
+    "subscriptionId": "bd794837-4d29-4647-9105-6339bfdb4e6a",
+    "resourceGroupName": "myRg",
+    "workspaceName": "myWorkspace",
+    "operationalInsightsResourceProvider": "Microsoft.OperationalIinsights",
+    "ThreatIntelligenceAppendTagsRequestBody": {
+      "ids": "f6b11318-1ca1-05cc-8499-20764752b567",
+      "threatIntelligenceTags": [
+        "tag1",
+        "tag2"
+      ]
+    }
+  },
+  "responses": {
+    "204": {}
```

Comment:
Can you also add "200": with example of appended tags
  "id": "/subscriptions/bd794837-4d29-4647-9105-6339bfdb4e6a/resourceGroups/myRg/providers/Microsoft.OperationalInsights/workspaces/myWorkspace/providers/Microsoft.SecurityInsights/ThreatIntelligence/180105c7-a28d-b1a2-4a78-234f6ec80fd6",
        "name": "180105c7-a28d-b1a2-4a78-234f6ec80fd6",
        "etag": "\"0000322c-0000-0800-0000-5e976c960000\"",
        "type": "Microsoft.SecurityInsights/ThreatIntelligence",
        "kind": "indicator",
        "properties": {
          "confidence": 78,
          "created": "2020-04-15T20:20:38.6160949Z",
          "createdByRef": "contoso@contoso.com",
          "externalId": "indicator--a2b6a95e-2108-4a38-bd49-ef95811bbcd7",
          "externalReferences": [],
          "granularMarkings": [],
          "lastUpdatedTimeUtc": "2020-04-15T20:20:38.6161887Z",
          "revoked": false,
          "source": "Azure Sentinel",
          "threatIntelligenceTags": [
            "tag1" , "tag2"
          ],
          "displayName": "new schema",
          "description": "debugging indicators",
          "threatTypes": [
            "compromised"
          ],
          "killChainPhases": [],
          "pattern": "[url:value = 'https://www.contoso.com']",
          "patternType": "url",
          "validFrom": "2020-04-15T17:44:00.114052Z"
        }
      }